bears-ansible-playbook.YML

```diff
@@ -0,0 +1,76 @@
+- name: Install To Do App on new web server
+  hosts: bearsapp
+  remote_user: ec2-user
+
+  vars_prompt:
+  - name: trello_api_key
+    prompt: Please provide your Trello API Key
+    private: Yes
+  - name: trello_token
+    prompt: Please provide your Trello Token
+    private: yes
+  - name: trello_board_id
+    prompt: Please provide your Trello_Board_ID
+    private: yes
+
+
+  tasks:
+  - name: install git
+    ansible.builtin.yum:
+            name: git
+            state: present
+    become: yes
+
+  - name: install Python
+    ansible.builtin.yum:
+            name: python3
+            state: present
+    become: yes
+
+  - name: install poetry
+    ansible.builtin.shell:
+            cmd: curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python3
+            creates: /root/.poetry/bin/poetry
+
+    become: yes
```

Comment:
You've overused `become` a bit - if you don't use it here, then poetry will install into `/home/ec2-user` and you can run quite a few of the subsequent commands without root